src/sockperf.cpp

```diff
@@ -3096,6 +3149,13 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
                 continue;
             }
 
+            if (tmp->server_addr.ss_family == AF_UNIX) {
+                s_user_params.tcp_nodelay = false;
+                strcpy(s_user_params.addr.addr_un.sun_path,tmp->server_addr.addr_un.sun_path);
+                s_user_params.addr.ss_family = AF_UNIX;
```

Comment:
why not copy the whole `tmp->server_addr`?

Reply:
agree, `tmp->server_addr` is copied